microsoft-edge/toc.yml

```
@@ -1067,9 +1067,12 @@
             - name: Define icons and a theme color
               href: progressive-web-apps-chromium/how-to/icon-theme-color.md
 
-            - name: Re-engage users with badges, notifications, and push messages
+            - name: Re-engage users with badges and notifications
               href: progressive-web-apps-chromium/how-to/notifications-badges.md
 
+            - name: Re-engage users with push messages
+              href: progressive-web-apps-chromium/how-to/push.md
```

```suggestion
              href: progressive-web-apps-chromium/how-to/push.md
              displayName: Re-engage users with badges, notifications, and push messages  # old top-of-page title
```
optional, this is a synonym Index entry: if you paste into upper left "Filter by title" box, will find article by using old title. see other instances of "displayName:" in toc file.